content/6_Advanced/Wavelet.problems.json

```diff
@@ -16,6 +16,43 @@
     }
   ],
   "wavelet": [
+    {
+      "uniqueId": "cf-840D",
+      "name": "Destiny",
+      "url": "https://codeforces.com/contest/840/problem/D",
+      "source": "CF",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Wavelet"],
+      "solutionMetadata": {
+        "kind": "autogen-label-from-site",
+        "site": "CF"
+      }
+    },
+    {
+      "uniqueId": "spoj-ILKQUERY2",
+      "name": "ILKQUERY2",
+      "url": "https://www.spoj.com/problems/ILKQUERY2/",
+      "source": "SPOJ",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Wavelet"],
+      "solutionMetadata": {
+        "kind": "none"
+      }
+    },
+    {
+      "uniqueId": "coci-2020-index",
```

Comment:
this problem already exists in the persistent data structure module

- rename id to coci-20-index

- keep tags consistent -- add pst to both?

this should fix ur build

Reply:
Should I also add wavelet to the one on persistent data structure?